src-ui/lib/components/BooksTableRow.svelte

```diff
@@ -34,8 +35,8 @@
 <!-- note: `slide` transitions (which I prefer here) don't currently work on tables: https://github.com/sveltejs/svelte/issues/4948 -->
 <tr transition:fade={{ duration: 300 }}>
   <td
-    ><button aria-label="delete book" class="delete" on:click={() => removeBook(book.id)}
-    ></button></td
+    ><Button aria-label="delete book" class="delete" onclick={() => removeBook(book.id)}
```

Comment:
Two changes:
1. Moved to using our new `Button` component
2. `on:click` is Svelte 4 syntax. `onclick` is Svelte 5.